game/templates/game/board.html

```
@@ -272,274 +330,188 @@
+            <p style="color: #aaa; margin-bottom: 30px; font-size: 0.95rem;">Enter names and select a game mode.</p>
 
-        .promo-title {
-            font-size: 1rem;
-            font-weight: 700;
-            color: #f0c040;
-            margin-bottom: 16px;
-            letter-spacing: 1px;
-            text-transform: uppercase;
-        }
+            <div id="nameInputs" style="display: flex; flex-direction: column; gap: 10px; margin-bottom: 25px;">
+                <input type="text" id="whiteNameInput" placeholder="White Player Name" maxlength="17" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)